README.md

```diff
@@ -1,41 +1,49 @@
 # BastCastle Simple Home Automation using Home Assistant
 
 ## The haHA Gateway Drug
-As a part of a cloud computing practicum, a team of six came together to build a home automation process using Home Assitant for three specific use cases: locks, curtains, and vents. We started testing with lights, so we kept those scipts in for funzies. Further, we built our process in the cloud but optimized it for migration to a PI-Hole. 
+As a part of a cloud computing practicum, a team of six came together to build a home automation
+process using Home Assistant for three specific use cases: locks, curtains, and vents. We started testing
+with lights, so we kept those scripts in for fun. Further, we built our process in the cloud within an AWS
+EC2 instance but optimized it for migration to a Pi-hole.
 
-Take our blood, sweat, and tears and build upon this foundation. Add addtional use cases of your own. At this point, in the world of home automation IoT, the sky is the limit: lights, locks, solar panels, televisions, alarams, cameras, vents, curtains, blinds, switches, sprinklers, etc. 
+Take our blood, sweat, and tears and build upon this foundation. Add additional use cases of your own.
+At this point, in the world of home automation IoT, the sky is the limit: lights, locks, solar panels,
+televisions, alarms, cameras, vents, curtains, blinds, switches, sprinklers, etc.
 
 WARNING: once you get started with haHA, you won't stop! 
 
-### Contributing Developers (should we link these to LI profiles?)
-- Alexi Most
-- Lucas Knezevich
-- Jason Beutler
-- Victor Navarro
-- Ashray Thapa
-- Brandon Dombrowsky
+### Contributing Developers
+- [Alexi Most](https://www.linkedin.com/in/alyoshenka/) 
```

Comment:
I'm on the fence about whether I want my LinkedIn profile here. Probably fine, but I want to give it a tad bit more consideration.